src/main/java/org/example/FlashcardManager.java

```diff
@@ -141,8 +134,8 @@ public void importF(String filename){
             }
         }else{
                 printLogMessage("Error");
-            }
         }
+    }
 
     public void hardestCard(){
```

Comment:
este metodo es muy largo, considera dividirlo en las funciones que hace, por ejemplo la busqueda e imprimir mensajes podrian ser metodos diferentes
```suggestion
    public void hardestCard() {
    if (flashcards.isEmpty()) {
        printLogMessage("There are no cards with errors.");
        return;
    }
    
    List<String> hardestCards = findHardestCards();
    printHardestCardsMessage(hardestCards);
}

private List<String> findHardestCards() {
    // Logic to find hardest cards
}

private void printHardestCardsMessage(List<String> hardestCards) {
    // Logic to print message
}
```